homeassistant/helpers/event.py

```diff
@@ -205,8 +206,16 @@ def async_track_state_change(
     being None, async_track_state_change_event should be used instead
     as it is slightly faster.
 
+    This function is deprecated and will be removed in Assistant 2025.5.
```

Comment:
nit: Just in case it's not intentional, you only wrote "Assistant" (instead of "Home Assistant" in multiple places).

Reply:
Its a mistake in all places

Reply:
Thanks for catching that

https://github.com/home-assistant/core/pull/115559